types/index.d.ts

```diff
@@ -289,10 +289,61 @@ declare namespace PhonegapPluginPush {
 		from?: string
 		notId?: string
 	}
-
+  /**
+   * Set Channel Variable,once created cannot be changed programmtically for importance and vibration
+   */
+  interface channel{
+    /**
+     * channel ID
+     */
+    id: string
+    /**
+     * camillebeaumont name and description fix
+     */
+    name: string 
+    /**
+     * camillebeaumont name and description fix,channel description
```

Comment:
In this case we also have official docs to copy from:
```suggestion
     * The user visible name of the channel. The recommended maximum length is 40 characters; the value may be truncated if it is too long.
```